packages/server/test/integration/http_requests_spec.coffee

```diff
@@ -1673,58 +1673,88 @@ describe "Routes", ->
               "Content-Type": "text/css"
             })
 
-      it "attaches auth headers when matches origin", ->
-        username = "u"
-        password = "p"
+      context "authorization", ->
+        it "attaches auth headers when matches origin", ->
+          username = "u"
+          password = "p"
 
-        base64 = Buffer.from(username + ":" + password).toString("base64")
+          base64 = Buffer.from(username + ":" + password).toString("base64")
 
-        @server._remoteAuth = {
-          username
-          password
-        }
+          @server._remoteAuth = {
+            username
+            password
+          }
 
-        nock("http://localhost:8080")
-        .get("/index")
-        .matchHeader("authorization", "Basic #{base64}")
-        .reply(200, "")
+          nock("http://localhost:8080")
+          .get("/index")
+          .matchHeader("authorization", "Basic #{base64}")
+          .reply(200, "")
 
-        @rp("http://localhost:8080/index")
-        .then (res) ->
-          expect(res.statusCode).to.eq(200)
+          @rp("http://localhost:8080/index")
+          .then (res) ->
+            expect(res.statusCode).to.eq(200)
 
-      it "does not attach auth headers when not matching origin", ->
-        nock("http://localhost:8080", {
-          badheaders: ['authorization']
-        })
-        .get("/index")
-        .reply(200, "")
+        it "does not attach auth headers when not matching origin", ->
+          nock("http://localhost:8080", {
+            badheaders: ['authorization']
+          })
+          .get("/index")
+          .reply(200, "")
 
-        @rp("http://localhost:8080/index")
-        .then (res) ->
-          expect(res.statusCode).to.eq(200)
+          @rp("http://localhost:8080/index")
+          .then (res) ->
+            expect(res.statusCode).to.eq(200)
 
-      it "does not modify existing auth headers when matching origin", ->
-        existing = "Basic asdf"
+        it "does not modify existing auth headers when matching origin", ->
+          existing = "Basic asdf"
 
-        @server._remoteAuth = {
-          username: "u"
-          password: "p"
-        }
+          @server._remoteAuth = {
+            username: "u"
+            password: "p"
+          }
 
-        nock("http://localhost:8080")
-        .get("/index")
-        .matchHeader("authorization", existing)
-        .reply(200, "")
+          nock("http://localhost:8080")
+          .get("/index")
+          .matchHeader("authorization", existing)
+          .reply(200, "")
```

Comment:
strange whitespace

Reply:
looks right to me, what am I missing?

Reply:
nvm, didn't see you had changed the scope